### ISSUE TITLE
Problems running boltz-gui

### ISSUE THREAD

**Issue by ghlopez** [ravishar313/boltz-gui#2]
Hello, 

I would like to ask you a question. I have been trying to run Boltz using the GUI and I have not been able to complete any of the examples as they all time out after 10 minutes. Other than that I do not get any other messages or errors.

I installed it in a conda environment (in case this could have an effect). I can confirm that Boltz can be successfully run in this environment in less than 10 minutes.

The hardware I am using is a M2 macbook pro with 96Gb or RAM. I noticed that in a post you indicated that running these examples took a couple of minutes in an m2 mac but at my end just running boltz takes longer than that (but less than 10 minutes for the time out).

Do you have any ideas about what might be happening?

Thanks in advance!

I have included the packages included in the environment and a couple of screenshots 

<img width="456" height="236" alt="Image" src="https://github.com/user-attachments/assets/5d0c6619-3bfe-472e-853e-4594e8d969a4" />

<img width="1061" height="520" alt="Image" src="https://github.com/user-attachments/assets/a6953e45-b91d-4fb3-9f75-d354a052ef86" />

# packages in environment at /miniconda3/envs/boltz2-gui:
#
# Name                    Version                   Build  Channel
aiohappyeyeballs          2.6.1                    pypi_0    pypi
aiohttp                   3.13.3                   pypi_0    pypi
aiosignal                 1.4.0                    pypi_0    pypi
antlr4-python3-runtime    4.9.3                    pypi_0    pypi
attrs                     25.4.0                   pypi_0    pypi
biopython                 1.84                     pypi_0    pypi
boltz                     2.2.1                    pypi_0    pypi
bzip2                     1.0.8                hd037594_9    conda-forge
ca-certificates           2026.1.4             hbd8a1cb_0    conda-forge
certifi                   2026.1.4                 pypi_0    pypi
charset-normalizer        3.4.4                    pypi_0    pypi
chembl-structure-pipeline 1.2.2                    pypi_0    pypi
click                     8.1.7                    pypi_0    pypi
dm-tree                   0.1.8                    pypi_0    pypi
docker-pycreds            0.4.0                    pypi_0    pypi
einops                    0.8.0                    pypi_0    pypi
einx                      0.3.0                    pypi_0    pypi
fairscale                 0.4.13                   pypi_0    pypi
filelock                  3.24.2                   pypi_0    pypi
frozendict                2.4.7                    pypi_0    pypi
frozenlist                1.8.0                    pypi_0    pypi
fsspec                    2026.2.0                 pypi_0    pypi
gemmi                     0.6.5                    pypi_0    pypi
gitdb                     4.0.12                   pypi_0    pypi
gitpython                 3.1.46                   pypi_0    pypi
hydra-core                1.3.2                    pypi_0    pypi
icu                       78.2                 h38cb7af_0    conda-forge
idna                      3.11                     pypi_0    pypi
ihm                       2.9                      pypi_0    pypi
jinja2                    3.1.6                    pypi_0    pypi
joblib                    1.5.3                    pypi_0    pypi
libexpat                  2.7.4                hf6b4638_0    conda-forge
libffi                    3.5.2                hcf2aa1b_0    conda-forge
liblzma                   5.8.2                h8088a28_0    conda-forge
libsqlite                 3.51.2               h1ae2325_0    conda-forge
libzlib                   1.3.1                h8359307_2    conda-forge
lightning                 2.6.1                    pypi_0    pypi
lightning-utilities       0.15.2                   pypi_0    pypi
llvmlite                  0.44.0                   pypi_0    pypi
markupsafe                3.0.3                    pypi_0    pypi
mashumaro                 3.14                     pypi_0    pypi
modelcif                  1.2                      pypi_0    pypi
mpmath                    1.3.0                    pypi_0    pypi
msgpack                   1.1.2                    pypi_0    pypi
multidict                 6.7.1                    pypi_0    pypi
ncurses                   6.5                  h5e97a16_3    conda-forge
networkx                  3.6.1                    pypi_0    pypi
nodejs                    0.1.1                    pypi_0    pypi
numba                     0.61.0                   pypi_0    pypi
numpy                     1.26.4                   pypi_0    pypi
omegaconf                 2.3.0                    pypi_0    pypi
openssl                   3.6.1                hd24854e_1    conda-forge
optional-django           0.1.0                    pypi_0    pypi
packaging                 26.0               pyhcf101f3_0    conda-forge
pandas                    3.0.1                    pypi_0    pypi
pillow                    12.1.1                   pypi_0    pypi
pip                       26.0.1             pyh8b19718_0    conda-forge
platformdirs              4.9.2                    pypi_0    pypi
propcache                 0.4.1                    pypi_0    pypi
protobuf                  5.29.6                   pypi_0    pypi
psutil                    7.2.2                    pypi_0    pypi
python                    3.12.12         h18782d2_2_cpython    conda-forge
python-dateutil           2.9.0.post0              pypi_0    pypi
pytorch-lightning         2.5.0                    pypi_0    pypi
pyyaml                    6.0.2                    pypi_0    pypi
rdkit                     2025.9.5                 pypi_0    pypi
readline                  8.3                  h46df422_0    conda-forge
requests                  2.32.3                   pypi_0    pypi
scikit-learn              1.6.1                    pypi_0    pypi
scipy                     1.13.1                   pypi_0    pypi
sentry-sdk                2.53.0                   pypi_0    pypi
setproctitle              1.3.7                    pypi_0    pypi
setuptools                82.0.0             pyh332efcf_0    conda-forge
six                       1.17.0                   pypi_0    pypi
smmap                     5.0.2                    pypi_0    pypi
sympy                     1.14.0                   pypi_0    pypi
threadpoolctl             3.6.0                    pypi_0    pypi
tk                        8.6.13               h010d191_3    conda-forge
torch                     2.10.0                   pypi_0    pypi
torchmetrics              1.8.2                    pypi_0    pypi
tqdm                      4.67.3                   pypi_0    pypi
types-requests            2.32.4.20260107          pypi_0    pypi
typing-extensions         4.15.0                   pypi_0    pypi
tzdata                    2025c                hc9c84f9_1    conda-forge
urllib3                   2.6.3                    pypi_0    pypi
uv                        0.10.4                   pypi_0    pypi
wandb                     0.18.7                   pypi_0    pypi
wheel                     0.46.3             pyhd8ed1ab_0    conda-forge
yarl                      1.22.0                   pypi_0    pypi
